Client/Frontend/Browser/TabManager.swift

```diff
@@ -851,8 +850,7 @@ extension TabManager {
                 withTry: { () -> Void in
```

Comment:
Maybe just use standard try/do?

Reply:
I dont think we can based on the comment above: `// This is wrapped in an Objective-C @try/@catch handler because NSKeyedUnarchiver may throw exceptions which Swift cannot handle`